packages/app/src/app/pages/Dashboard/Content/routes/Recent/RecentContent.tsx

```diff
@@ -23,10 +21,7 @@ import { RecentHeader } from './RecentHeader';
 
 const StyledWrapper = styled(Stack)`
   width: 100%;
-  max-width: ${GRID_MAX_WIDTH}px;
-  padding: 0 calc(2 * ${GUTTER}px) 64px;
   overflow: auto;
-  margin: 28px auto 0;
```

Comment:
The goal of the margin-top was to make it align with the workspace picker:

![Screen Shot 2023-02-01 at 22 21 08](https://user-images.githubusercontent.com/24959348/216207772-5e7faed6-2a1d-49f5-88c9-a95ce09fbfa0.png)